app/src/main/res/layout/screen_scan_simple.xml

```diff
@@ -27,4 +30,27 @@
     app:layout_constraintTop_toTopOf="parent"
     app:srcCompat="@drawable/ic_scan_qr_viewfinder" />
 
+  <androidx.cardview.widget.CardView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="8dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginStart="8dp"
+    android:background="@color/white100"
+    android:visibility="gone"
+    app:layout_constraintBottom_toBottomOf="@+id/qrCodeScannerView"
+    app:layout_constraintEnd_toEndOf="@+id/qrCodeScannerView"
+    app:layout_constraintStart_toStartOf="parent">
+
+    <EditText
+      android:id="@+id/editText"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_gravity="center"
+      android:ems="10"
+      android:hint="123 4567"
+      android:imeOptions="actionGo"
```

Comment:
Shouldn't this be `actionSearch`? We won't know until the search has completed whether the target exists or not.

Reply:
Good catch.